### DISCUSSION TITLE
📜-Réglement

### DISCUSSION THREAD

**Discussion by Ulysse-srfc24** [Web3Invest/.github#12] [General]
Bonjour a tous, j'ouvre ce fil de discussion a propos de la créations d'un nouveaux réglement pour W3i :)

**Comment by badmingtoncedric** [comment]
>  [Modèle](https://www.service-public.fr/particuliers/vosdroits/R2707) fourni par le service-public

# Règlement intérieur de l’association xxxxx
> Adopté par l’assemblée générale du jj/mm/aaaa

__**Attention**__ : Il ne s’agit là que d’un exemple. La rédaction du règlement intérieur doit être adaptée à
chaque association en fonction de ses activités réelles et surtout de ses statuts, notamment lorsqu’il
convient de compléter et expliciter ces statuts.

## Article 1 – Agrément des nouveaux membres.
Tout nouveau membre doit être parrainé et présenté par deux membres de l’association, dont au
moins un membre fondateur, préalablement à son agrément.
Il est agréé par le conseil statuant à la majorité de tous ses membres.
Le conseil statue lors de chacune de ses réunions sur les demandes d’admission présentées.
Les personnes désirant adhérer doivent remplir un bulletin d’adhésion.

## Article 2 – Démission – Exclusion – Décès d’un membre
- 1. La démission doit être adressée au président du conseil par lettre recommandée. Elle n’a pas
à être motivée par le membre démissionnaire.
- 2. Comme indiqué à l’article « N » des statuts, l’exclusion d’un membre peut être prononcée par
le conseil, pour motif grave. Sont notamment réputés constituer des motifs graves :
  - la non-participation aux activités de l’association ;
  - une condamnation pénale pour crime et délit ;
  - toute action de nature à porter préjudice, directement ou indirectement, aux activités de
l’association ou à sa réputation.
En tout état de cause, l’intéressé doit être mis en mesure de présenter sa défense,
préalablement à la décision d’exclusion.
La décision d’exclusion est adoptée par le conseil statuant à la majorité des deux tiers des
membres présents.
- 3. En cas de décès d’un membre, les héritiers ou les légataires ne peuvent prétendre à un
quelconque maintien dans l’association.
La cotisation versée à l’association est définitivement acquise, même cas en cas de démission,
d’exclusion, ou de décès d’un membre en cours d’année.

## Article 3 – Assemblées générales – Modalités applicables aux votes
- 1. Votes des membres présents
Les membres présents votent à main levée. Toutefois, un scrutin secret peut être demandé par le
conseil ou « X » % (par exemple, 20%) des membres présents.
- 2. Votes par procuration
Comme indiqué à l’article « N » des statuts, si un membre de l’association ne peut assister
personnellement à une assemblée, il peut s’y faire représenter par un mandataire dans les conditions
indiquées audit article (« ou ne peut pas »).

## Article 4 – Indemnités de remboursement.
Seuls les administrateurs et/ou membres élus du bureau, peuvent prétendre au remboursement des
fais engagés dans le cadre de leurs fonctions et sur justifications. Préciser un tarif maximum de
nuitée, repas, un % de facture téléphonique, etc.) Prévoir la possibilité d’abandon de ces
remboursements et d’en faire don à l’association en vue de la réduction d’impôt sur le revenu art. 200
du CGI).

## Article 5 – Commission de travail.
Des commissions de travail peuvent être constituées par décision du conseil d’administration.

## Article 6 – Modification du règlement intérieur
Le présent règlement intérieur pourra être modifié par le conseil ou par l’assemblée générale ordinaire
à la majorité (simple ou par exemple des deux tiers) des membres.

**Comment by badmingtoncedric** [comment]
Voici un second modèle sur lequel s'appuyer et je pense que ChatGPT peut être un bon outil pour réaliser ce travail

# Règlement Intérieur

## Article 1 : Force obligatoire
Le règlement intérieur a la même force obligatoire pour tous les membres que les
statuts de l’association. Nul ne peut s’y soustraire puisqu’il est accepté lors de
l’adhésion.

## Article 2 : Modalités d’adhésion et de désistement
### Adhésion
Les adhérents de la saison précédente sont prioritaires pour le renouvellement de
leur inscription pour la saison suivante. La date limite des réinscriptions est fixée par
le Comité directeur. Les nouvelles demandes d’inscription sont prises en compte en
deux temps :
- Une phase de préinscriptions a lieu lors de la « Journée des Associations »
organisée en septembre par la ville de Vincennes. Les demandes d’adhésion
y sont enregistrées dans la limite des places disponibles (en fonction du
nombre maximum de membres défini par le Comité directeur moins les
réinscriptions).
- Une fois la demande enregistrée, l’adhésion est effective lorsque le dossier
d’inscription complet a été réceptionné par le Comité directeur et que la
cotisation a été réglée. À l’exception des compétiteurs classés, nul ne peut
valablement être inscrit sans avoir été au préalable préinscrit.

### Désistement
Seuls les désistements demandés dans les 7 jours qui suivent la réception du
dossier complet seront tolérés (le dossier d’inscription sera alors restitué dans son
intégralité). Passé ce délai, aucun désistement ne sera accepté.

## Article 3 : Cotisation / Licence – Assurance
### Cotisation
Le montant de la cotisation comprend :
- celui de la licence et de l’assurance,
- celui de l’adhésion.
Le montant de la cotisation est annuel et valable pour une saison (du 1er septembre
au 31 août de l’année suivante).
En cours de saison, il ne sera fait droit à aucune demande de remboursement,
même partielle et quels qu’en soient les motifs (manque d’assiduité, blessure, congé
maternité, mutation professionnelle, etc).

### Licence – Assurance
Les adhérents sont tous licenciés. Les licenciés reçoivent directement à leur domicile
leur licence sur la base des informations communiquées lors de leur inscription. Ils
bénéficient, à ce titre, d’une assurance les couvrant en cas d’accident. Les
dispositions relatives à cette assurance figurent dans le formulaire de prise de
licence.
En cas d’accident, une déclaration auprès de l’assureur doit se faire dans les 5 jours
qui suivent la survenance du dommage. L’adhérent devra en parallèle informer le
Comité directeur par email.

## Article 4 : Discipline
Tenue et accès aux terrains
Le badminton est un sport qui se pratique en salle. En conséquence, il est obligatoire
de se présenter sur les terrains muni d’une paire de chaussures de sport de salle.
Chaque adhérent doit également veiller à adopter une tenue de sport correcte et
décente. Les vêtements et chaussures de ville doivent être laissés dans les
vestiaires mis à disposition dans chaque gymnase. Seuls les effets personnels de
valeur (sacs à main, clés, téléphones portables, portefeuilles, etc.) peuvent être
emportés dans la salle à condition de ne pas empiéter sur les espaces de jeu.
Respect des règles de fonctionnement des gymnases
Chaque adhérent doit se conformer aux règles de fonctionnement des gymnases. Il
est en particulier rappelé qu’il est interdit :
- de pénétrer sur les terrains avec des chaussures portées à l’extérieur, avec
des vélos, trottinettes, rollers, etc,
- d’y introduire de l’alcool,
- de pratiquer sur les terrains toute autre activité que le badminton,
- de donner accès aux terrains à des personnes extérieures au club.

### État d’esprit
L’association respecte l’esprit sportif. Tout adhérent s’engage donc à entretenir
gaieté, loyauté et respect des autres. Tout propos anti-sportif, injurieux, sexiste ou
raciste se verra sanctionné par le Comité directeur.

### Sanctions
Le non respect du présent règlement ou toute faute grave d’une autre nature que
celle évoquée ci-dessus peut entraîner la suspension à titre temporaire ou l’exclusion
définitive de l’adhérent, après mise en garde par le Comité directeur.

## Article 5 : Encadrement des mineurs
Les mineurs sont encadrés par un entraîneur ou un responsable de l’association
durant les créneaux réservés aux « jeunes » et sont donc sous leur responsabilité.
En revanche, en dehors de ces créneaux, ils restent sous la totale responsabilité de
leur(s) parent(s) ou représentant légal. Il est donc demandé aux parents de vérifier la
présence de l’entraîneur ou d’un responsable du club en début de séance.
Toute adhésion « jeunes » entraîne l’acceptation de toutes les clauses du présent
règlement. Tout manquement à l’une des clauses, toute action périlleuse,
perturbatrice ou contestataire sera immédiatement sanctionné par un avertissement
oral, puis, en cas de récidive, par une exclusion temporaire ou définitive, décidée par
le Comité directeur.

## Article 6 : Responsabilité
L’association décline toute responsabilité en cas d’accident ou de détérioration du
matériel, si l’un des membres utilise d’autres installations sportives que celles
destinées à la pratique du badminton.
L’association décline toute responsabilité en cas d’incidents ou d’accidents survenus
en dehors de l’enceinte des gymnases.
L’association décline toute responsabilité en cas de perte ou de vol sur les terrains
ou dans l’enceinte des gymnases.

## Article 7 : Habilitation
Tous les adhérents de l’association sont habilités à faire respecter le présent
règlement. En revanche, seuls les membres du Comité directeur sont habilités à
prononcer l’exclusion temporaire ou définitive de tout adhérent ayant contrevenu
délibérément au présent règlement.

## Article 8 : Assemblée générale
Tout adhérent est informé par email de la tenue des Assemblées générales (date,
lieu et ordre du jour). Il s’engage et s’efforcera d’y assister pour approuver l’exercice
précédent et élire les membres du Comité directeur selon les modalités définies par
les statuts de l’association. En cas d’empêchement, l’adhérent peut se faire
représenter par un autre adhérent de l’association.

## Article 9 : Modification et réclamation
Le présent règlement pourra être modifié lors de l’Assemblée générale annuelle ou
exceptionnellement par le Comité directeur. Toute réclamation pourra être adressée
par écrit au président.

**Comment by Ulysse-srfc24** [comment]
Super c'est niquel, Merci nous allons regarder sa, et voir comment l'arranger pour W3i ! :)

**Comment by siak631** [comment]
Article 1 : Objet et Mission

1.1 L'Association Web3Invest a pour mission d'éduquer le public sur les principes fondamentaux du Web3, d'organiser des ateliers, des conférences et des formations pour favoriser la compréhension et l'adoption des technologies Web3, de faciliter la mise en réseau et la collaboration entre les passionnés et les professionnels du Web3, ainsi que de promouvoir des projets et des initiatives Web3 innovants au sein de la communauté francophone.

Article 2 : Adhésion

2.1 Les membres de l'association sont tenus de payer une cotisation annuelle de 100 euros pour bénéficier du statut de membre VIP si ils ne font pas parties des 1000 premiers adhérents.

Article 3 : Statut de Membre VIP

3.1 Les membres VIP bénéficient des avantages suivants :

Accompagnement par un coach expérimenté dans le domaine du Web3.
Accès exclusif aux ateliers, conférences et formations organisés par l'association.
Participation aux projets open source de l'association.
Accès au serveur Discord de l'association pour les échanges et la collaboration avec les autres membres.
3.2 Le paiement de la cotisation annuelle de 100 euros confirme le statut de membre VIP. En spécifiant le fait que les 1000 premiers adhérents ayant payés le pass VIP se verront avoir ce droit à vie.  

Article 4 : Respect sur le Serveur Discord

4.1 Le respect des règles et des autres membres sur le serveur Discord de l'association est indispensable. Tout comportement contraire aux valeurs de respect, de tolérance et de collaboration sera sanctionné.

4.2 Les membres sont encouragés à contribuer de manière positive à la communauté et à partager leurs connaissances et leur expertise.

Article 5 : Participation active

5.1 Les membres de l'association sont encouragés à suivre le repository GitHub de l'association pour rester informés des dernières mises à jour et participer aux projets open source.

5.2 Les membres sont également encouragés à participer activement aux événements, ateliers et conférences organisés par l'association afin d'approfondir leurs connaissances et de rencontrer d'autres enthousiastes du Web3.

5.3 Les membres sont invités à partager les missions de l'association avec leur réseau personnel afin d'aider à atteindre un public plus large et à promouvoir les objectifs de Web3Invest.

Article 6 : Modification du Règlement

6.1 Toute modification du présent règlement intérieur devra être soumise à l'approbation de l'Assemblée Générale des membres de l'association.


Voila une version 1.0 du règlement. n'hésitez pas a me faire part de votre ressenti (et me dire quelles modifications st nécesaires)

Content de participer au projet et longue vie a w3i !!!!
